.eslintrc

```diff
@@ -1,3 +1,13 @@
 {
-    "extends": "ebay"
-}
\ No newline at end of file
+    "extends": "ebay",
+    "rules": {
+         "import/order": [2, {"groups": ["builtin", "external", "internal", "parent", "sibling", "index"]}],
```

Comment:
For `import/order`, do we have to add this as a dev dependency?

https://github.com/benmosher/eslint-plugin-import

Reply:
Yes @CestDiego . Lets come with order if you have any thought